res/css/views/elements/_Field.scss

```diff
@@ -115,3 +115,36 @@ limitations under the License.
     background-color: $field-focused-label-bg-color;
     color: $greyed-fg-color;
 }
+
+.mx_Field_valid input,
+.mx_Field_valid select,
+.mx_Field_valid textarea {
+    border-color: $input-valid-border-color ! important;
```

Comment:
What triggered all the `important`? Would adding an extra selector like `.mx_Field.mx_Field_valid` avoid the need for it?

Reply:
it's so that the valid/invalid state clobbers the focus-selected colour. i can't remember the specificity of :focus, hence clobbering with `!important`, which i agree is ugly. someone with more css-fu would be welcome to switch the selectors to be more elegant ;)